readthedocs/core/permissions.py

```diff
@@ -10,9 +9,29 @@
 class AdminPermissionBase:
 
     @classmethod
-    def admins(cls, obj):
+    def owners(cls, obj):
+        """
+        Return the owners of `obj`.
+
+        If organizations are enabled,
+        we return the owners of the project organization instead.
+        """
+        from readthedocs.organizations.models import Organization
         from readthedocs.projects.models import Project
+
+        if isinstance(obj, Project):
+            if settings.RTD_ALLOW_ORGANIZATIONS:
+                obj = obj.organizations.first()
+            else:
+                return obj.users.all()
+
+        if isinstance(obj, Organization):
+            return obj.owners.all()
```

Comment:
If it's a project _and_ ALLOW_ORGANIZATION this method returns `None`.

Reply:
`obj` is re-assigned below to be the project's organization